ugvc/comparison/concordance_utils.py

```diff
@@ -39,10 +45,16 @@ def read_hdf(file_name: str, key: str = "all", skip_keys: collections.abc.Iterab
         for k in skip_keys:
             if k in keys:
                 keys.remove(k)
-        dfs = [pd.read_hdf(file_name, key=key) for key in keys]
+        dfs = []
+        for k in keys:
+            tmpdf = pd.read_hdf(file_name, key=k)
+            if columns_subset is not None:
+                tmpdf = tmpdf[[x for x in columns_subset if x in tmpdf.columns]]
+            if tmpdf.shape[0] > 0:
+                dfs.append(tmpdf)
         return pd.concat(dfs)
     if key == "all_human_chrs":
-        dfs = [pd.read_hdf(file_name, key=f"chr{x}") for x in list(range(1, 23)) + ["X"]]
+        dfs = [pd.read_hdf(file_name, key=f"chr{x}") for x in list(range(1, 23)) + ["X", "Y"]]
```

Comment:
Is this going to work for hg19?

Reply:
No, but we're also not using this key.
I added "all_hg19_human_chrs", since I don't know if anyone uses this key already.